Protocols/Http/Request.php

```diff
@@ -504,7 +504,7 @@ protected function parseUploadFiles($http_post_boundary)
                 switch ($header_key) {
                     case "content-disposition":
                         // Is file data.
-                        if (\preg_match('/name="(.*?)"; filename="(.*?)"$/i', $header_value, $match)) {
```

Comment:
Why remove `$` ?

Reply:
Request header value:
```
form-data; name="upload"; filename="测.png"; filename*=UTF-8''%E6%B5%8B.png
```
If there's $，Matching is null.
```
\preg_match('/name="(.*?)"; filename="(.*?)"$/i', $header_value, $match))
```